mitodl/odl-video-service
# Added Sentry configuration

#### What are the relevant tickets?
closes #67 

#### What's this PR do?
Adds sentry configuration

#### How should this be manually tested?
add the sentry ins to your env. Make raise an exception and check that it appears on Sentry
